test/functional/mining_basic.py

```diff
@@ -37,9 +40,23 @@ def assert_template(node, block, expect, rehash=True):
 class MiningTest(BitcoinTestFramework):
     def set_test_params(self):
         self.num_nodes = 2
-        self.setup_clean_chain = False
+        self.setup_clean_chain = True
+
+    def mine_chain(self):
+        self.log.info('Create some old blocks')
+        for _ in range(0, 200):
```

Comment:
not sure why we changed this logic

Reply:
Have to bump mocktime on all nodes because otherwise other nodes will ban us for relaying blocks from the future.